scripts/conf.py

```diff
@@ -388,13 +383,10 @@ def install_jobsubmit_lua(lkp: util.Lookup) -> None:
         for part in lkp.cfg.partitions.values()
         for tpu_nodeset in part.partition_nodeset_tpu
     ):
-        conf_options = NSDict(
-            {
-                "scripts_dir": lkp.cfg.slurm_scripts_dir or dirs.scripts,
-            }
-        )
-        conf_resp = blob_get("slurm-tpl-job-submit-lua").download_as_text()
-        conf = conf_resp.format(**conf_options)
+        conf_options = {
```

Comment:
Can we also add an option to pass job_submit.lua in other cases, than when TPU partition is provided? To allow customers to manage easily this customization?

Reply:
Definetly, but this needs to be done on toolkit side. The `slurm_files` module already accepts `job_submit_lua_tpl` variable. Is that what you were interested in?

Reply:
Yes, slurm_files accept job_sumbit_lua_tpl, but `conf.py` installs it only when you have TPU partition defined (IIUC above condition)